app-text/sigil/sigil-0.9.16.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit cmake-utils python-single-r1 readme.gentoo-r1 xdg-utils
+
+MY_P="Sigil-${PV}"
+
+DESCRIPTION="Sigil is a multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/"
+SRC_URI="https://github.com/Sigil-Ebook/Sigil/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
just use `${P}` for the tarball name here

Reply:
Done.